elide-async/src/main/java/com/yahoo/elide/async/service/DefaultResultStorageEngine.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.async.service;
+
+import static com.yahoo.elide.core.EntityDictionary.NO_VERSION;
+
+import com.yahoo.elide.Elide;
+import com.yahoo.elide.async.models.AsyncQueryResultStorage;
+import com.yahoo.elide.core.DataStore;
+import com.yahoo.elide.core.DataStoreTransaction;
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.RequestScope;
+import com.yahoo.elide.core.filter.dialect.RSQLFilterDialect;
+import com.yahoo.elide.jsonapi.models.JsonApiDocument;
+import com.yahoo.elide.request.EntityProjection;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.sql.Blob;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.UUID;
+
+import javax.inject.Singleton;
+import javax.sql.rowset.serial.SerialBlob;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+
+@Singleton
+@Slf4j
+@Getter
+public class DefaultResultStorageEngine implements ResultStorageEngine {
+
+    @Setter private Elide elide;
+    @Setter private DataStore dataStore;
+    private EntityDictionary dictionary;
+    private RSQLFilterDialect filterParser;
+
+    public DefaultResultStorageEngine() {
+    }
+
+    public DefaultResultStorageEngine(Elide elide, DataStore dataStore) {
+        this.elide = elide;
+        this.dataStore = dataStore;
+        dictionary = elide.getElideSettings().getDictionary();
+        filterParser = new RSQLFilterDialect(dictionary);
+    }
+
+    @Override
+    public AsyncQueryResultStorage storeResults(UUID asyncQueryID, String responseBody) {
+        byte[] temp = responseBody.getBytes();
+        Blob response = null;
+        try {
+            response = new SerialBlob(temp);
+        } catch (SQLException e) {
+            log.error("Exception: {}", e);
+        }
+        AsyncQueryResultStorage asyncQueryResultStorage = new AsyncQueryResultStorage();
+        asyncQueryResultStorage.setId(asyncQueryID);
+        asyncQueryResultStorage.setResult(response);
+
+        URL url = null;
+        try {
+            url = new URL("https://elide.io/asyncQueryID");
```

Comment:
url should be constructed using three parts base_url, "/asyncQueryResultStore/" , asyncQueryID. base_url can be initialized by setting in the constructor. For elide standalone, baseUrl can be another Settings and in spring we can pass it in application.yaml